### ISSUE TITLE
Questions about the preprocessing for data

### ISSUE THREAD

**Issue by leVirve** [StanfordVL/taskonomy#4]
Hello,
I have some questions on some arguments that I cannot find the detail in the documentation.

For the data loading in this function, what's the meaning of the `fixated` condition, and how could we get this mean/std? Is this pre-calculated from the dataset?

https://github.com/StanfordVL/taskonomy/blob/fee83d022424bf1d3c40f840ae1d528f221587ed/taskbank/lib/data/load_ops.py#L752-L761

And, when should we flip the XY-axis, what will happen if not flip in room layout task by your comment in this line? 🤔

https://github.com/StanfordVL/taskonomy/blob/fee83d022424bf1d3c40f840ae1d528f221587ed/taskbank/lib/data/load_ops.py#L1120-L1131

**Comment by alexsax** [comment]
Thanks for the detailed questions with inline code!
> For the data loading in this function, what's the meaning of the fixated condition, and how could we get this mean/std? Is this pre-calculated from the dataset?

The `fixated` camera pose task takes in two images where the optical center of the image "fixates" on the same physical point. We also have a `nonfixated` task where the optical centers do not have this restriction. The mean/std are computed from the training set. We are looking to release the data shortly. 

> And, when should we flip the XY-axis, what will happen if not flip in room layout task by your comment in this line? 🤔

This line is just to make the camera matrix orented correctly. While making the paper, we worked with a couple different conventions. This is because python 2 and python 3 label use different conventions for the X-Y axis in an image. 

The short answer is: The code should work as-is. Any misalignments would be pretty obvious since they entail a 90-degree rotation!
